### ISSUE TITLE
Custom docker daemon args is not working

### ISSUE THREAD

**Issue by shaoye** [gitpod-io/gitpod#10844]
### Bug description

Got Permission denied error following the steps in #8435 

```
1. Add environment variable DOCKER_DAEMON_ARGS to your user settings. The content should look like this:
{ "remap-user": "1000" } (Means container id 1000 will be mapped to gitpod user).
2. Start workspace
3. mkdir /workspace/tmp/me
4. Run docker image which supports running as unprivileged user e.g. sudo docker run -it -u node -v /workspace/tmp/me:/tmp node sh
5. Create file within container e.g. touch /tmp/foo
6. Verify that the created file has gitpod:gitpod permissions in the workspace
```

got permission denied at step 5, 
```
touch: cannot touch '/tmp/foo': Permission denied
```
the `node` user can't create any files since `/tmp` is owned by `gitpod` user
```
/tmp$ ls -al
total 4
drwxr-xr-x 2 33333 33333   40 Jun 21 21:59 .
drwxr-xr-x 1 root  root  4096 Jun 21 22:00 ..
```

### Steps to reproduce

as above

**Comment by utam0k** [comment]
Hi, @shaoye. I investigated this issue and found the name of the env was wrong.
You set `DOCKER_DAEMON_ARGS`, but `DOCKERD_ARGS` is the right name.  
https://www.gitpod.io/docs/environment-variables#user-specific-environment-variables

However, the name of the environment variable in the PR summary was incorrect. I have corrected that as well. Thanks.
![image](https://user-images.githubusercontent.com/13010913/177086601-c45bb864-5912-4336-8ff6-8727c1d10a33.png)
